op-service/client/bss_client.go

```diff
@@ -1,274 +1 @@
 package client
```

Comment:
Can we delete this file instead of leaving it empty?

Reply:
yup 👍